lib/lightning/work_orders.ex

```diff
@@ -394,13 +394,7 @@ defmodule Lightning.WorkOrders do
          }
        }) do
     edges
-    |> Enum.reduce(Graph.new(), fn edge, graph ->
-      graph
-      |> Graph.add_edge(
-        edge.source_trigger_id || edge.source_job_id,
-        edge.target_job_id
-      )
-    end)
+    |> Enum.reduce(Graph.new(), &Graph.add_edge(&2, &1))
```

Comment:
Could we maintain the long `fn` block here? 😅 the capture operator can  get confusing real quick. I think just using `edge` and `graph` makes it really easy to know what's going on.

Reply:
Perfect! 🙌 
Generally, I only use the short form when it's one argument. The moment it's more than 1, it gets confusing for me, even for `reduce` 😅